backend/internal/test/integration/serverexperimentation/cmd/envoyconfiggen/main.go

```diff
@@ -0,0 +1,112 @@
+package main
```

Comment:
a lot of copy-pasting for integrations tests - not a perfect solution but I think it should be just fine for our use case for now.